xbmc/GUIInfoManager.cpp

```diff
@@ -5832,6 +5884,18 @@ bool CGUIInfoManager::ConditionsChangedValues(const std::map<INFO::InfoPtr, bool
   return false;
 }
 
+bool CGUIInfoManager::GetShowVideoOSD() const
+{
+  CGUIWindow *pOSD = g_windowManager.GetWindow(WINDOW_DIALOG_VIDEO_OSD);
+  return pOSD && pOSD->IsDialogRunning();
+}
+
+bool CGUIInfoManager::GetShowMusicOSD() const
```

Comment:
you can use `CGUIWindowManager::IsWindowActive(int id)` instead of introducing these methods.